Libraries/Animated/__tests__/AnimatedMock-test.js

```diff
@@ -49,4 +56,76 @@ describe('Animated Mock', () => {
     });
     done();
   });
+
+  const itCallsStartCallback = () => {
+    jest.runAllTimers();
+
+    expect(mockStartCallback).toHaveBeenCalledTimes(1);
+  };
+
+  describe('Animated.parallel', () => {
+    it('calls the start callback', () => {
+      AnimatedMock.parallel([]).start(mockStartCallback);
+
+      jest.runAllTimers();
```

Comment:
Why do we call runAllTimers twice? Do we need to call it in the test if its part of `itCallsStartCallback`?